doc/src/minimize.rst

```diff
@@ -32,30 +32,49 @@ Description
 Perform an energy minimization of the system, by iteratively adjusting
 atom coordinates.  Iterations are terminated when one of the stopping
 criteria is satisfied.  At that point the configuration will hopefully
-be in local potential energy minimum.  More precisely, the
+be in a local potential energy minimum.  More precisely, the
 configuration should approximate a critical point for the objective
 function (see below), which may or may not be a local minimum.
 
-The minimization algorithm used is set by the
-:doc:`min_style <min_style>` command.  Other options are set by the
-:doc:`min_modify <min_modify>` command.  Minimize commands can be
-interspersed with :doc:`run <run>` commands to alternate between
-relaxation and dynamics.  The minimizers bound the distance atoms move
-in one iteration, so that you can relax systems with highly overlapped
-atoms (large energies and forces) by pushing the atoms off of each
-other.
-
-Alternate means of relaxing a system are to run dynamics with a small
-or :doc:`limited timestep <fix_nve_limit>`.  Or dynamics can be run
-using :doc:`fix viscous <fix_viscous>` to impose a damping force that
-slowly drains all kinetic energy from the system.  The :doc:`pair_style soft <pair_soft>` potential can be used to un-overlap atoms while
-running dynamics.
-un-overlap atoms while running dynamics.
+The minimization algorithm used is set by the :doc:`min_style
+<min_style>` command.  Other options are set by the :doc:`min_modify
+<min_modify>` command.  Minimize commands can be interspersed with
+:doc:`run <run>` commands to alternate between relaxation and dynamics.
+The minimizers bound the distance atoms may move in one iteration, so
+that you can relax systems with highly overlapped atoms (large energies
+and forces) by pushing the atoms off of each other.
+
+.. admonition:: Neighbor list update settings
+   :class: note
+
+   The distance that atoms can move during individual minimization steps
+   can be quite large, especially at the beginning of a minimization.
+   Thus `neighbor list settings <neigh_modify>` of *every = 1* and
+   *delay = 0* are **required**.  This may be combined with *check = no*
+   (always update the neighbor list) or *check = yes* (only update the
+   neighbor list if at least one atom has moved more than half the
+   `neighbor list skin <neighbor>` distance since the last
+   reneighboring).  Using *check = yes* is recommended since it avoids
+   unneeded reneighboring steps when the system is closer to the minimum
+   and thus atoms move only small distances.
```

Comment:
Add a sentence about why you might want to use check = no ?
Only for debugging in some context ?  I think that if every=1, the
utility of check = no is minimal.  Because you will (over time) get different
answers for check = no/yes, due to re-ordering atoms.

Reply:
done